project/ScalafixBuild.scala

```diff
@@ -184,7 +184,10 @@ object ScalafixBuild extends AutoPlugin with GhpagesKeys {
     Compile / unmanagedSourceDirectories ++= {
       val sourceDir = (Compile / sourceDirectory).value
       CrossVersion.partialVersion(scalaVersion.value) match {
-        case Some((2, n)) if n >= 12 => Seq(sourceDir / "scala-2.12+")
```

Comment:
I still see files under these directories, so the removal of this makes them dead code (and probably compilation errors). This avoid duplication between 2.12/2.13 source directories, so why not keep it?